src/test/java/com/epam/training/auction_backend/services/AuctionsServiceImplTestIT.java

```diff
@@ -0,0 +1,105 @@
+package com.epam.training.auction_backend.services;
+
+import com.epam.training.auction.common.AuctionTransferObject;
+import com.epam.training.auction.common.UsersService;
+import com.epam.training.auction_backend.entity.Auction;
+import com.epam.training.auction_backend.entity.User;
+import com.epam.training.auction_backend.util.CommonIntegrationTestSuit;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import javax.transaction.Transactional;
+import java.util.List;
+
+public class AuctionsServiceImplTestIT extends CommonIntegrationTestSuit {
+
+    @Autowired
+    private AuctionRepository auctionRepository;
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private UsersService usersService;
+
+    private AuctionsServiceImpl auctionsService;
+
+    @Before
+    public void setUp() {
+        auctionsService = new AuctionsServiceImpl(auctionRepository);
+    }
+
+    @Test
+    @Transactional
+    public void getArchivedAuctions() {
+        final int expectedArchivedAuctionsSize = 2;
+
+        // create seller and winner for auctions
```

Comment:
To understand this test you need reed 7 lines of code or 3 lines of comments so I think we need here comments to reduce reading